tests/integration/targets/ndo_mcp_global_policy/tasks/main.yml

```diff
@@ -0,0 +1,300 @@
+          - cm_update_mcp_global_policy.previous.enablePduPerVlan == nm_update_mcp_global_policy.previous.enablePduPerVlan == false
+          - cm_update_mcp_global_policy.previous.key is undefined 
+          - nm_update_mcp_global_policy.previous.key is undefined
+          - cm_update_mcp_global_policy.previous.loopDetectMultFactor == nm_update_mcp_global_policy.previous.loopDetectMultFactor == 3
+          - cm_update_mcp_global_policy.previous.protectPortDisable == nm_update_mcp_global_policy.previous.protectPortDisable == true
+          - cm_update_mcp_global_policy.previous.txFreq == nm_update_mcp_global_policy.previous.txFreq == 2
+          - cm_update_mcp_global_policy.previous.txFreqMsec == nm_update_mcp_global_policy.previous.txFreqMsec == 0
+          - cm_update_mcp_global_policy.current == cm_update_mcp_global_policy.proposed
+          - cm_update_mcp_global_policy.current.name == nm_update_mcp_global_policy.current.name == "ansible_mcp_global_policy_new"
+          - cm_update_mcp_global_policy.current.adminState == nm_update_mcp_global_policy.current.adminState == "enabled"
+          - cm_update_mcp_global_policy.current.description == nm_update_mcp_global_policy.current.description == "A Global MCP Policy"
+          - cm_update_mcp_global_policy.current.enablePduPerVlan == nm_update_mcp_global_policy.current.enablePduPerVlan == true
+          - cm_update_mcp_global_policy.current.key == nm_update_mcp_global_policy.current.key == "cisco"
+          - cm_update_mcp_global_policy.current.loopDetectMultFactor == nm_update_mcp_global_policy.current.loopDetectMultFactor == 4
+          - cm_update_mcp_global_policy.current.protectPortDisable == nm_update_mcp_global_policy.current.protectPortDisable == false
+          - cm_update_mcp_global_policy.current.txFreq == nm_update_mcp_global_policy.current.txFreq == 3
+          - cm_update_mcp_global_policy.current.txFreqMsec == nm_update_mcp_global_policy.current.txFreqMsec == 10
+          - update_mcp_global_policy_again is not changed
+          - update_mcp_global_policy_again.previous == update_mcp_global_policy_again.current
+          - nm_update_mcp_global_policy_no_uuid is failed
+          - nm_update_mcp_global_policy_no_uuid.msg == "MCP Global Policy with the UUID{{":"}} 'bad-uuid' not found"
+
+    # QUERY
+    - name: Query the MCP Global Policy with name
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        name: ansible_mcp_global_policy_new
+        state: query
+      register: query_mcp_policy_with_name
+
+    - name: Query the MCP Global Policy with UUID
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        uuid: '{{update_mcp_global_policy_again.current.uuid}}'
+        state: query
+      register: query_mcp_policy_with_uuid
+
+    - name: Query the MCP Global Policy without name or UUID
```

Comment:
I think in other we have been doing a query with both name and UUID provided, could you add that?

Reply:
Would it be okay to link to an example test that queries with both name and UUID?
The closest I can see is [this recent one](https://github.com/CiscoDevNet/ansible-mso/pull/550/files#diff-643a3396f501aad132095eb0e57a7b43569d3241d48b1cea7cb77b740013d3b7R228) which uses `name: '{{ fakevar | default(omit)}}'`, since `fakevar` doesn't exist it will omit name.

Reply:
Ok think this is the one that I meant, so not relevant in that case.